linux/snowagent.config

```diff
@@ -27,72 +27,112 @@
+		  <FileSystem>cifs</FileSystem>
+		  <FileSystem>devpts</FileSystem>
+		  <FileSystem>devtmpfs</FileSystem>
+		  <FileSystem>fuse</FileSystem>
+		  <FileSystem>fuse.gvfsd-fuse</FileSystem>
+		  <FileSystem>fuse.vmware-vmblock</FileSystem>
+		  <FileSystem>fuseblk</FileSystem>
+		  <FileSystem>nfs</FileSystem>
+		  <FileSystem>nfs4</FileSystem>
+		  <FileSystem>panfs</FileSystem>
+		  <FileSystem>proc</FileSystem>
+		  <FileSystem>rpc-pipes</FileSystem>
+		  <FileSystem>smbfs</FileSystem>
+		  <FileSystem>sysfs</FileSystem>
+		  <FileSystem>tmpfs</FileSystem>
+		  <FileSystem>vmblock</FileSystem>
+		  <FileSystem>vmhgfs</FileSystem>
+		  <!-- Exclude common temporary directories -->
+		  <Path>/tmp*</Path>
+		  <Path>/var/tmp*</Path>
+		  <Path>/var/log*</Path>
+		  <!-- Exclude Oracle audit files -->
+		  <Path>*/oracle/*/audit*</Path>
+		  <!-- Exclude JDE and DB -->
+		  <Path>/jde*</Path>
+		  <Path>/redo*</Path>
+		  <Path>/undo*</Path>
+		  <Path>/archive*</Path>
+		  <Path>/trn*</Path>
+		  <Path>/cjde*</Path>
+		  <Path>/distbi*</Path>
+		  <Path>/mfgbi*</Path>
+		  <!-- Exclude common file extensions -->
+		  <Path>*.au</Path>
+		  <Path>*.bmp</Path>
+		  <Path>*.bmu</Path>
+		  <Path>*.cfg</Path>
+		  <Path>*.class</Path>
+		  <Path>*.conf</Path>
+		  <Path>*.csm</Path>
+		  <Path>*.css</Path>
+		  <Path>*.dic</Path>
+		  <Path>*.enc</Path>
+		  <Path>*.gif</Path>
+		  <Path>*.h</Path>
+		  <Path>*.htm</Path>
+		  <Path>*.html</Path>
+		  <Path>*.jpg</Path>
+		  <Path>*.js</Path>
+		  <Path>*.log</Path>
+		  <Path>*.mo</Path>
+		  <Path>*.mof</Path>
+		  <Path>*.packlist</Path>
+		  <Path>*.pcf</Path>
+		  <Path>*.pc</Path>
+		  <Path>*.pf</Path>
+		  <Path>*.pl</Path>
+		  <Path>*.png</Path>
+		  <Path>*.po</Path>
+		  <Path>*.rdf</Path>
+		  <Path>*.sdl</Path>
+		  <Path>*.so</Path>
+		  <Path>*.sql</Path>
+		  <Path>*.ttf</Path>
+		  <Path>*.txe</Path>
+		  <Path>*.utf8</Path>		  
+		  <Path>*.xpt</Path>
+		  <Path>*.zip</Path>
+		  <!-- *.xml *.txt and *.properties files would normally be excluded but are required for SIOMS -->
+		  <!--<Path>*.xml</Path>
+		  <Path>*.txt</Path>
+		  <Path>*.properties</Path>-->
         </Exclude>
     </Software>
     <!-- Scanning for Oracle databases, requires SIOS  -->
-    <!-- <Oracle enabled="true">
-        <DefaultInstanceCredentials>
-            <UserName></UserName>
-            <Password></Password>
-        </DefaultInstanceCredentials>
+    <Oracle enabled="false">
+		<!-- Password needs to be encrypted-->
+        <!--<DefaultInstanceCredentials>
+            <UserName></UserName>			
+            <Password></Password> 
+        </DefaultInstanceCredentials>-->
         <Include>
             <AllInstances>true</AllInstances>
         </Include>
-    </Oracle> -->
+    </Oracle>
     <Logging>
-        <MaxSize>10240</MaxSize>
+        <MaxSize>2048</MaxSize>
         <!-- error, warning, info, trace, verbose -->
-        <Level>verbose</Level>
+        <Level>warning</Level>
```

Comment:
I see your point, but running production with verbose logging is not ideal.
Althought we recycle the files to avoid build up, writting to file is "expensive" and can contribute to customer complaints on agent performance.

Reply:
Have we actually had such complaints? If that is a thing then we can adjust the logging library to do buffered writes instead.

Reply:
If we have no actual complaints and have no actual facts pointing to it being a problem, why make support harder?

Reply:
I would argue, that for a recommended configuration, we don't want to tell customers to run in Verbose.
No production application recommends logs to be set to Verbose/Debug. Those are only used for troubleshooting.
Also, and that is my fault too, we need to be consistent with our messaging.
In this page we are actually recommending "error" for daily use.
https://docs.snowsoftware.io/technical-videos/snow-atlas/package-builder/snow-inventory-agent-config-file/
We also have "error" as the default setting in the agent.
https://docs.snowsoftware.io/other-snow-products/snow-inventory-agents-and-oracle-scanners/agent-configuration-file/#logging

I agree that it would help support, or not since it will have multiple runs and stuff that is not relevant, but if we want to set our recommendation to verbose, we need to be consistent about it and explain our reasoning, as it is not standard practice on other software.

Reply:
Per, what did you see in your experience that makes you think having the agent running in verbose all the time helps at?
You mention response time, as we always ask for a verbose log in support. Do you have any information that backs that up?
Do we have a logging issue that we are trying to solve with verbose?
If the support engineer is going going to troubleshoot anyway, are we really making a difference with verbose?
Is our agent really that problematic that we need verbose all the time? If so, shouldn't we focus on fixing the underlying issue instead?

Reply:
@gpcarval Errors logs just the error itself, not what led up to it. I haven't documented all the times when we've asked for verbose logs, but feel free to ask support if they feel verbose logging gives additional information or not.

And yes, the logging issue is that there is an option to not log everything. imho.

Reply:
@gpcarval how to handle errors or warnings on such airtight systems, where people are not allowed to simply change the config and run the agent on demand or even access the system? I assume the costs in additional storage in a compressed snowpack is less then getting a proper log in the second run. People are literally using backup features, to have snowpacks at hand if support asks for such thing.

As this is just templates, just leave them at verbose. Can be reduced from the field, in alignment with what the customer thinks, his storage costs, vs the labour manually changing it on demand.